CONTRIBUTING.md

```diff
@@ -9,6 +9,9 @@ We are happy to accept contributions. To accept a contribution, we require two t
   https://www.clahub.com/agreements/Pelagicore/softwarecontainer
   - We don't like CLAs but we have to have one. We'll remove it as soon as we can.
 
+We try to follow the [https://wiki.gnome.org/Git/CommitMessages](GNOME guidelines) for commit
```

Comment:
I do agree with Kursat. My suggestion is to just rephrase this to "We aim to follow".